packages/react-provider/src/index.tsx

```diff
@@ -63,6 +66,28 @@ export const CourierProvider: React.FunctionComponent<ICourierContext> = ({
     middleware,
   });
 
+  useEffect(() => {
+    if (_transport) {
+      // this means the transport was passed in and we shouldn't subscribe
```

Comment:
what are the scenarios where one would pass in transport?

Reply:
that was originally how it all worked, you had to create and pass in the transport.  so we have many storybook examples doing it this way.  im not sure if it will happen in the wild... but it could